Prog/Tabulate householder population for projection.sas

```diff
@@ -0,0 +1,247 @@
+    3 = "All Other ";
+  value hispan
+     .n = 'Not available'
+    0 = 'Not Hispanic'
+    1 = 'Hispanic';
+ value racenew
+   .n = 'Not available'
+    1 = 'White non-Hispanic'
+    2 = 'Black non-Hispanic'
+    3 = "Hispanic "
+	4 = "All other non-Hispanic ";
+  value agegroup
+     .n = 'Not available'
+    1= "0-5 years old"
+	2= "5-9 years old"
+	3= "10-14 years old "
+	4="15-19 years old"
+	5="20-24 years old "
+	6="25-29 years old"
+	7="30-34 years old"
+	8="35-39 years old"
+	9="40-44 years old"
+    10="45-49 years old"
+    11= "50-54 years old"
+    12= "55-59 years old"
+    13= "60-64 years old"
+    14= "65-69 years old"
+    15= "70-74 years old"
+    16= "75-79 years old" 
+    17 = "80-84 years old"
+    18= "85+ years old";
+run;
+
+
+%macro householdinfo(year);
+
+
+	data Household_&year. ;
+		set Ipums.Acs_&year._NC;
+
+		%assign_NCcounty;
+		%assign_NCcounty;
+		county_char = put(county, 5.);
+	run;
+
+	data Inc_&year. ;
+	set NCHsg.IncomeLimits_&year. (where= (State=37));
+        county_new = put(County,z3.);
+        state2= put(State, z2.);
+        county_char= state2||county_new;
+	run;
+
+	proc sort data= Household_&year. ;
+	by county_char;
+	run;
+
+	proc sort data= Inc_&year. ;
+	by county_char;
+	run;
+
+	data Household_&year._2 ;
+	merge Household_&year. Inc_&year.;
+	by county_char ;
+	run;
+
+	data Householddetail_&year.;
+		set Household_&year._2;
+		keep race hispan age hhincome hhincome_a pernum relate gq county hhwt perwt year serial numprec race1 agegroup hud_inc totpop_&year. l50_1- l50_8 l80_1- l80_8  median&year.   ;
+
+		 %Hud_inc_NCHsg( hhinc=hhincome_a, hhsize=numprec )
+		  label
+		  hud_inc = 'HUD income category for household'; 
+
+		  /*
+		if HHINCOME in ( 9999999, .n , . ) then incomecat=.;
+		else do; 
+		    if HHINCOME<=32600 then incomecat=1;
+			else if 32600<HHINCOME<=54300 then incomecat=2;
+			else if 54300<HHINCOME<=70150 then incomecat=3;
+			else if 70150<HHINCOME<=108600 then incomecat=4;
+			else if 108600<HHINCOME<=130320 then incomecat=5;
+			else if 130320<HHINCOME<=217200 then incomecat=6;
+			else if 217200 < HHINCOME then incomecat=7;
+		end;
+*/
+		if hispan=0 then do;
+
+		 if race=1 then race1=1;
+		 else if race=2 then race1=2;
+		 else race1=4;
+		end;
+
+		if hispan in(1 2 3 4) then race1=3;
+
+		if 0<=age<25 then agegroup=1;
+		else if 25<=age<45 then agegroup=2;
+		else if 45<=age<65 then agegroup=3;
+		else if age>=65 then agegroup=4;
+
+		totpop_&year. = 1;
+		run;
+
+		proc freq data=Householddetail_&year.;
+		  tables race1 * agegroup  / list missing;
+		run;
+
+		proc sort data=Householddetail_&year.;
+		by county2 agegroup race1 relate;
+		run;
+
+%mend householdinfo;
+
+%householdinfo(2013);
+%householdinfo(2014);
+%householdinfo(2015);
+%householdinfo(2016);
+%householdinfo(2017);
+
+
+/*change serial 560493 and 255313 in 2013 from HoH to other nonrelative as they really reflect GQ (GQ=5) and those households have 11 and 20 people (not related)*/
```

Comment:
you need this step.  there are 7 households with gq=5 that really are group homes.  please reassign to other nonrelative.